Fedora/README.md

```diff
@@ -0,0 +1,10 @@
+#Fedora
+![Fedora_LOGO](Imgs/Fedora_LOGO.png)
+---
+##Distribution History
+---
+Fedora Linux or ___"Fedora"___ is an **Open_source Linux** distribution that was released in **6 November 2003**. It was originally developed as a continuation of [Red Hat Linux](https://en.wikipedia.org/wiki/Red_Hat_Linux) project.
+Although fedora is not as stable as Red Hat Enterprise Linux (RHEL), but it is completely free to use and full of innovations, unlike Red Hat that is typically sold through an annual subscription.
+
+##Distro Image
```

Comment:
Space here too

Reply:
Note: The changes are minor and I'm not sure whether I should've merged and requested fixes later, but I'll play it safe